doc/doc-requirements.txt

```diff
@@ -1,6 +1,5 @@
 Cython>=0.24
 # Frozen Sphinx requirements for easier pip installation
-sphinx==1.7.9
```

Comment:
if we don't need this line, maybe also remove the commented line before as well?

Reply:
I initially thought the same when removing that line, but looks like that comment was inserted in https://github.com/kivy/kivy/pull/2959/files so it doesn't belong to the line just removed

Reply:
Ah indeed, well spotted.